src/screen/HtmlScreen.js

```diff
@@ -96,8 +96,15 @@ class HtmlScreen extends RequestScreen {
 	copyNodeAttributesFromContent_(content, node) {
 		content = content.replace(/[<]\s*html/ig, '<senna');
 		content = content.replace(/\/html\s*\>/ig, '/senna>');
-		node.innerHTML = content;
-		var placeholder = node.querySelector('senna');
+		if (UA.isIe) {
```

Comment:
Using `var` here makes a weird use of hoisting and might create issues with the code below if we actually switch to `let` instead.

Might be better to do:

```
let placeholder;

if (UA.isIe) {
    const tempNode = globals.document.createReange().createContextualFragment(content);
    placeholder = tempNode.querySelector('senna');
}
else {
    node.innerHTML = content;
    placeholder = node.querySelector('senna');
}

if (placeholder) {
    // `placeholder` will have at least been defined
}
```

Reply:
Sure, about `var` usage on Senna.js, we need clarify this utilisation because in some senna classes we are using `var` instead `let` and `const`.

Reply:
Yeah, mind opening an issue in the repo to standardize this? Code is quite old here, not sure how useful would it be to try to change all the code at once :)